plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -356,37 +350,6 @@ public void testShowCreateTable()
                         getLocationForTable(bucketName, "person")));
     }
 
-    @Test
-    public void testInputDataSize()
```

Comment:
> Remove BaseDeltaLakeConnectorSmokeTest.testInputDataSize


Why removed?
where is equivalent being tested?

Reply:
https://github.com/trinodb/trino/pull/17556

Reply:
This test doesn't seem useful, and certainly not worth the complexity of needing to configure the Hive plugin just for this one single test.